StripeFinancialConnections/StripeFinancialConnections/Source/Native/Shared/GenericInfoScreen/GenericInfoBodyView.swift

```diff
@@ -88,6 +87,45 @@ private func TextBodyEntryView(
     return textView
 }
 
+private func ImageBodyEntryView(
+    _ imageBodyEntry: FinancialConnectionsGenericInfoScreen.Body.ImageBodyEntry
+) -> UIView? {
+    guard let imageUrlString = imageBodyEntry.image.default else {
+        return nil
+    }
+    let imageView = AutoResizableImageView()
+    imageView.setImage(with: imageUrlString)
+    return imageView
+}
+
+// `UIImageView` that will autoresize itself to be
+// full width, but maintain aspect ratio height
+private class AutoResizableImageView: UIImageView {
+
+    override var intrinsicContentSize: CGSize {
+        if let image, image.size.width > 0 {
+            let height = image.size.height * (bounds.width / image.size.width)
+            return CGSize(
+                width: UIView.noIntrinsicMetric,
+                height: height
+            )
+        } else {
+            return CGSize(
+                width: bounds.width,
+                height: 200 // give some height with assumption that an image will load
```

Comment:
this was taken from Stripe.js CSS...this is just there as a temporary value to reduce any big jump once the image does load